composer.json

```diff
@@ -29,11 +29,11 @@
         "php": ">=5.4",
         "doctrine/common": "~2.4",
         "doctrine/orm": "~2.4",
-        "symfony/http-kernel": "~2.4",
-        "symfony/dependency-injection": "~2.4"
+        "symfony/http-kernel": "~2.4|~3.0",
+        "symfony/dependency-injection": "~2.4|~3.0"
     },
     "require-dev": {
-        "symfony/framework-bundle": "~2.4, >=2.4.8",
```

Comment:
I think ~ is same ^ just preferring different higher to lower. But not sure, let me check composer docs.